src/web/badge/Cargo.toml

```diff
@@ -6,10 +6,8 @@ authors = ["Onur Aslan <onur@onur.im>"]
 license-file = "LICENSE"
 repository = "https://github.com/rust-lang/docs.rs"
 documentation = "https://docs.rs/badge"
-
-[lib]
-path = "badge.rs"
+edition = "2018"
 
 [dependencies]
-base64 = "0.9.0"
-rusttype = "0.7"
```

Comment:
Please do not update dependencies in a refactor PR. You can make a separate PR for this.